src/main/scala/ObservingMode.scala

```diff
@@ -0,0 +1,28 @@
+package basic
+
+import gem.enum._
+import gem.math._
+
+sealed trait ObservingMode
+
+sealed trait SpectroscopyMode extends ObservingMode {
+  def fieldOfView:        Offset     // x, y .. lame?
+  def resolution:         Double     // dimensionless
```

Comment:
I don't understand resolution yet.  How can it be dimensionless?  3000 of what?

Reply:
I don't get it either. It's on the list of things to figure out.